vernac/declaremods.ml

```diff
@@ -718,7 +718,7 @@ let start_module export id args res fs =
   let () = Global.push_context_set ~strict:true ctx in
   openmod_info := { cur_typ = res_entry_o; cur_typs = subtyps };
   let prefix = Lib.start_module export id mp fs in
-  Nametab.(push_dir (Until 1) (prefix.obj_dir) (GlobDirRef.DirOpenModule prefix));
+  Nametab.(push_dir (Until 1) (prefix.obj_dir) (GlobDirRef.DirOpenModule mp));
```

Comment:
Given this change, wouldn't it make sense to also modify `Lib.start_module` so that it directly returns `obj_dir`? At this point, I don't think there is anyone left who cares about the actual prefix.